### PR TITLE
Editorial updates to RESTful APIs

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

Initial editorial updates based on provided sample codes. 

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/references/http_api_reference.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ curl --request DELETE \
...
+- 200 Success.
 
 ```
 {
     "error_code": 0
 }
 ```
 
-- 500 Internal Service Error
+- 500 Internal service error.
```
Comment (writinwaters) [inline docs/references/http_api_reference.md:71]: Why in some cases 500 mean error only?

File: docs/references/http_api_reference.md [hunk 2]
```diff
@@ -764,7 +764,7 @@ curl --request GET \
 
 ## Show variables
 
-Gets variables.
+Get variables.
```
Comment (writinwaters) [inline docs/references/http_api_reference.md:767]: Needs further clairification.

File: docs/references/http_api_reference.md [hunk 3]
```diff
@@ -808,7 +808,7 @@ curl --request GET \
 
 #### Response
 
-- 200 success
+- 200 Success.
```
Comment (writinwaters) [inline docs/references/http_api_reference.md:811]: No error response?

File: docs/references/http_api_reference.md (hunk 1 above)
Comment (JinHai-CN) [reply docs/references/http_api_reference.md:71]: This is HTTP status code means error happens in server.

File: docs/references/http_api_reference.md (hunk 2 above)
Comment (JinHai-CN) [reply docs/references/http_api_reference.md:767]: Yes, variables in system.

Comment (JinHai-CN) [reply docs/references/http_api_reference.md:767]: These variables indicate the status of infinity. And one more document are needed to clarify these variables.

File: docs/references/http_api_reference.md (hunk 1 above)
Comment (writinwaters) [reply docs/references/http_api_reference.md:71]: 500 Error.

**Pull request merged**